opdracht-tech-it-easy/src/main/java/nl/novi/opdrachttechiteasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,54 @@
+package nl.novi.opdrachttechiteasy.controllers;
+
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@RequestMapping("/televisions")
+@RestController
+public class TelevisionController {
+
+
+    @GetMapping()
+    public ResponseEntity<String> television(){
+        return ResponseEntity.ok("television");
+    }
+
+    @GetMapping("/type")
+    public ResponseEntity<String> television(@RequestParam String type){
+        return ResponseEntity.ok("Television " + type);
+    }
+
+    @PostMapping("")
+    public ResponseEntity<String> addTelevisions(@RequestBody String television){
+        System.out.println(television);
+        return ResponseEntity.created(null).build();
```

Comment:
Je returned in deze ResponseEntity geen body, maar de returnvalue van je methode zegt wel dat er een String body gereturned wordt (`ResponseEntity<String>`). Je kunt een ".body(television)" tussen je ".created" en ".build" zetten. Maar als je geen body terug wilt sturen, kun je ook de returnvalue van de methode aanpassen naar `ResponseEntity<Void>`